crates/oxc_transformer/src/options/transformer.rs

```diff
@@ -89,7 +89,10 @@ impl TransformOptions {
             },
             es2016: ES2016Options { exponentiation_operator: true },
             es2018: ES2018Options { object_rest_spread: Some(ObjectRestSpreadOptions::default()) },
-            es2017: ES2017Options { async_to_generator: true },
+            es2017: ES2017Options {
+                // Turned off because it is not ready.
+                async_to_generator: true,
```

Comment:
Shouldn't this be `false`?

```suggestion
                async_to_generator: false,
```